### PR TITLE
Refactored shared checkbox styles

### PR DESCRIPTION
**Pull request by ronan-f**
Clean up before adding another checkbox to quizzing.

Refactored shared checkbox styles following the pattern [documented by LitElement ](https://lit-element.polymer-project.org/guide/styles#sharing-styles).

### CODE REVIEW

File: components/d2l-activity-editor/styles/checkbox-styles.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	:host {
+		display: block;
+	}
+
+	:host([hidden]) {
+		display: none;
+	}
+
+	d2l-input-checkbox {
+		margin: 0;
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/styles/checkbox-styles.js:13]: I've seen `margin: 0` isn't on some of our editors probably when Vincent and I missed it when we put them in.

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-pager-and-alerts-editor.js [hunk 2]
```diff
@@ -10,26 +11,7 @@ class ActivityQuizDisablePagerAndAlertsEditor
...
-			:host {
-				display: block;
-			}
-
-			:host([hidden]) {
-				display: none;
-			}
-
-			d2l-input-checkbox {
-				padding-right: 1rem;
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-pager-and-alerts-editor.js:24]: No `margin: 0` was here

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-anonymous-marking-editor.js [hunk 3]
```diff
@@ -16,28 +17,12 @@ class ActivityAssignmentAnonymousMarkingEditor
...
-			:host([hidden]) {
-				display: none;
-			}
-
 			.d2l-body-small {
 				margin: 0 0 0.3rem 0;
 			}
 
-			d2l-input-checkbox {
-				padding-right: 1rem;
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-anonymous-marking-editor.js:33]: No margin: 0 was here

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js [hunk 4]
```diff
@@ -10,26 +11,7 @@ class ActivityQuizDisableRightClickEditor
...
-			:host {
-				display: block;
-			}
-
-			:host([hidden]) {
-				display: none;
-			}
-
-			d2l-input-checkbox {
-				padding-right: 1rem;
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js:24]: No margin: 0 was here

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-hints-editor.js [hunk 5]
```diff
@@ -10,26 +11,7 @@ class ActivityQuizHintsEditor
...
-			:host {
-				display: block;
-			}
-
-			:host([hidden]) {
-				display: none;
-			}
-
-			d2l-input-checkbox {
-				padding-right: 1rem;
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-hints-editor.js:24]: No margin: 0 was here

File: components/d2l-activity-editor/styles/checkbox-styles.js (hunk 1 above)
Comment (mostlyfabulous) [reply components/d2l-activity-editor/styles/checkbox-styles.js:13]: Don't know if that property affects anything being `0` and all

Comment (ronan-f) [reply components/d2l-activity-editor/styles/checkbox-styles.js]: Good point. I updated it to remove the `margin: 0` from the shared styles and only implement it where it was explicitly designed before.

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-pager-and-alerts-editor.js (hunk 2 above)
Comment (ronan-f) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-pager-and-alerts-editor.js:24]: Updated

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-anonymous-marking-editor.js (hunk 3 above)
Comment (ronan-f) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-anonymous-marking-editor.js:33]: Updated

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-hints-editor.js (hunk 5 above)
Comment (ronan-f) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-hints-editor.js:24]: Updated

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js (hunk 4 above)
Comment (ronan-f) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js:24]: Updated

**Review by mostlyfabulous — APPROVED** [review]
LGTM

**Pull request merged**